_data/simple-icons.json

```
@@ -10282,12 +10282,13 @@
         {
             "title": "QuantConnect",
             "hex": "F5AE29",
-            "source": "https://www.quantconnect.com/docs/home/home"
+            "source": "https://www.quantconnect.com",
+            "guidelines": "https://www.quantconnect.com/terms"
```

```suggestion
            "source": "https://www.quantconnect.com/docs/home/home"
```

Let's revert this change - and have a separate issue to track this one.